### ISSUE TITLE
MethodNotAllowedException on /auth/ldap route

### ISSUE THREAD

**Issue by cdknight** [tituspijean/flarum-ext-auth-ldap#13]
When I install this extension, I get a 405 error on the route /auth/ldap when I try to test my LDAP settings and log in.

Using debug mode still doesn't alleviate the problem and gives no information.

Any help is appreciated with this problem.

**Comment by tituspijean** [comment]
The extension is still not up to date with latest beta. I guess this is the issue. I have no time during the next 10 days to have a look into it.

**Comment by cdknight** [comment]
I know a bit of Laravel, I will be looking at it after I try another way of getting LDAP to work with a different extension

**Comment by tituspijean** [comment]
If you know Laravel, you can try to update all the namespaces of this extension to match the last beta's namespaces. I will most welcome any PR you can make. :)

**Comment by cdknight** [comment]
I started to update the namespaces, but I am very confused on how to set up a good flarum dev environment on either my main Ubuntu workstation or Debian server that is running Flarum. I got as far as creating a local packages repository and rsyncing the modified extension directory to my server (I couldn't get flarum to work locally because of permission errors or something like that). I will be working on setting up a better environment, if you have any suggestions let me know (in general too for developing a flarum extension).
